python/ray/autoscaler/_private/util.py

```diff
@@ -724,6 +724,18 @@ def filter_placement_group_from_bundle(bundle: ResourceBundle):
     return demand_lines
 
 
+def get_constraints_report(lm_summary: LoadMetricsSummary):
+    constraint_lines = []
+    for bundle, count in lm_summary.request_demand:
+        line = f" {bundle}: {count}+ from request_resources()"
```

Comment:
actually what does the `+` sign after count mean? We should know the exact count

Reply:
I guess it was originally trying to align with the style of other demand texts. We indeed have the exact count. I will remove the `+`.